.github/instructions/SqlServerDsc-guidelines.instructions.md

```
@@ -3,7 +3,16 @@ description: SqlServerDsc-specific guidelines for AI development.
 applyTo: "**"
 ---
 
-# SqlServerDsc Guidelines
+# SqlServerDsc Requirements
+
+## Build & Test Workflow Requirements
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)